resources/extruders/cartesio_extruder_2.def.json

```diff
@@ -14,7 +14,7 @@
             "maximum_value": "3"
         },
         "machine_nozzle_offset_x": { "default_value": 0.0 },
-        "machine_nozzle_offset_y": { "default_value": 60.0 },
+        "machine_nozzle_offset_y": { "default_value": 0.0 },
```

Comment:
Are you sure about this? This way the disallowed area's are not displaced when different extruders are used.